src/main/java/edu/greenblitz/tobyDetermined/subsystems/IntakeObjSensor.java

```diff
@@ -0,0 +1,78 @@
+package edu.greenblitz.tobyDetermined.subsystems;
+
+import com.revrobotics.ColorMatch;
+import com.revrobotics.ColorMatchResult;
+import com.revrobotics.ColorSensorV3;
+import edu.greenblitz.tobyDetermined.RobotMap;
+import edu.greenblitz.utils.GBCommand;
+import edu.wpi.first.wpilibj.I2C;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj.util.Color;
+
+public class IntakeObjSensor extends GBSubsystem {
```

Comment:
Obj is not that clear, maybe IntakeSensor or IntakeGameObjectSensor
Also, maybe just add this to the intake subsystem

Reply:
now called IntakeGameObjectSensor, i think its easy to understand now